### PR TITLE
Add CircleCI config

### PR DESCRIPTION
**Pull request by openjck**
Co-Authored-By: William Lachance <wrlach@gmail.com>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+jobs:
+  build-and-test:
+    executor:
+      name: node/default
+      tag: "12.16"
+    steps:
+      - checkout
+      - node/install-packages
+      - run: npm install
+      - run: npx run lint
```
Comment (hamilton) [inline .circleci/config.yml:13]: Should this be `npm run lint`? I don't know if the `npx` command does anything differently here or if it is functionally the same.

Comment (openjck) [reply .circleci/config.yml:13]: Yes it should be, thank you!

**Comment by openjck** [comment]
Updated

**Comment by openjck** [comment]
Actually, don't merge this yet. We don't actually have an `npm run lint` script. 😄 

I'm working on that now.

**Comment by openjck** [comment]
We can merge this after #10 is merged

**Pull request merged**